src/Actions/ManagesContacts.php

```diff
@@ -83,6 +101,36 @@ public function findOrCreateContact($email, $firstName, $lastName, $orgid = null
         return $this->createContact($email, $firstName, $lastName, $orgid);
     }
 
+    /**
+     * Updates a contact.
+     *
+     * @param Contact|int|string $id
+     * @param string|null        $email
+     * @param string|null        $firstName
+     * @param string|null        $lastName
+     * @param null               $orgid
+     *
+     * @return Contact|null
+     */
+    public function updateContact($id, $email, $firstName, $lastName, $orgid = null)
+    {
+        $id = $this->getContactId($id);
+
+        $this->put('contacts/'.$id, ['json' => ['contact' => compact('email', 'firstName', 'lastName', 'orgid')]]);
+
+        return $this->findContactById($id);
+    }
+
+    /**
+     * Deletes a contact.
+     *
+     * @param Contact|int|string $contact
+     */
+    public function deleteContact($contact)
```

Comment:
Can you make it strong typed? `deleteContact(Contact $contact)`

Reply:
Good spot.

Reply:
... actually - [see comment below](https://github.com/testmonitor/activecampaign/pull/17#discussion_r314461289)